File: Lab2/Python/main.py
from module import *

firstName = "first.dat"
secondName = "second.dat"
inputFile(firstName)
showFile(firstName)
createSecondFile(firstName, secondName)
print("Second file: ")
showFile(secondName)
editFile(secondName)
print("\nEdited file: ")
showFile(secondName)

File: Lab2/C++/main.cpp
#include "header.h"

int main() {
    const char* first = "first.dat";
    const char* second = "second.dat";
    input_file(first);
    cout << "First file: " << endl;
    output_file(first);
    create_second_file(first, second);
    cout << "Second file: " << endl;
    output_file(second);
    edit_file(second);
    cout << "Edited file: " << endl;
    output_file(second);
    return 0;
}
